lib/internal/fs.js

```diff
@@ -52,7 +53,7 @@ function stringToFlags(flag) {
     case 'xa+': return O_APPEND | O_CREAT | O_RDWR | O_EXCL;
   }
 
-  throw new Error('Unknown file open flag: ' + flag);
+  throw new errors.Error('ERR_UNK_FILE_OPEN_FLAG', flag);
```

Comment:
I second.
IMHO the special case is little bit counterproductive:
```js
try {
  // do some stuff
  var f = fs.openSync('g.txt', 'rgh');
  // do other stuff
catch (e) {
  if (e.code == 'ERR_INVALID_OPT_VALUE') ; // swallow because I'm a special kind of guy
  else throw e;
}
```